### ISSUE TITLE
Allow changing task while editing time entry

### ISSUE THREAD

**Issue by bits-fritz** [Hoglet/TimeIT#59]
It would nice to have the task changed, while editing a time entry.
This is especially a good thing, while editing a split entry. If I split an entry this is almost done because of working on different tasks following each other or in parallel.
The attached picture might tell more specific.

Thanks for providing timeit :-
![edit-time](https://user-images.githubusercontent.com/20949248/149820261-58b0493c-1d57-46e0-bef3-f432563d9d00.png)
)

**Comment by Hoglet** [comment]
Are we talking about:
1) adding a new time to another task directly after adding current time-slice?
or:
2) editing a time-slice moving it to another task
or:
3) adding an identical time slice to two tasks?

**Comment by bits-fritz** [comment]
We talk about:

>     2. editing a time-slice moving it to another task

This one :o))

Thanks for Your attention..

**Comment by Hoglet** [comment]
Sounds reasonable.

**Comment by Hoglet** [comment]
Feel free to contribute. I might take long time to get to it.
